### DISCUSSION TITLE
Please give a copy of the SMTP configuration details.

### DISCUSSION THREAD

**Discussion by moeik** [outline/outline#3579] [Self Hosting]
I was going to use outlook mail as a mail service, but it never worked.

```
# To support sending outgoing transactional emails such as "document updated" or
# "you've been invited" you'll need to provide authentication for an SMTP server
SMTP_HOST=smtp.office365.com
SMTP_PORT=587
SMTP_USERNAME=i@xxx
SMTP_PASSWORD=pass
SMTP_FROM_EMAIL=i@xxx
SMTP_REPLY_EMAIL=i@xxx
SMTP_TLS_CIPHERS=high
SMTP_SECURE=true
```

**Comment by a-gerhard** [comment]
Same issue here with same configuration. Another service running on the same machine is doing fine though, so it definitely is a config issue.

`{"error":"139772373715760:error:1408F10B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:331:\n","stack":"Error: 139772373715760:error:1408F10B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:331:\n","level":"error","message":"Error processing task in EmailTask"}`

**Comment by ChinaeduO** [comment]
Any updates on this?

**Comment by LanyonConrad** [comment]
Here's mine:

SMTP_HOST=smtp-mail.outlook.com
SMTP_PORT=587
SMTP_USERNAME=username@companydomain.com
SMTP_PASSWORD=password
SMTP_FROM_EMAIL=username@companydomain.com
SMTP_TLS_CIPHERS=TLSv1.3
SMTP_SECURE=false

@a-gerhard I think that was the error I was getting before adding SMTP_TLS_CIPHERS=TLSv1.3

**Reply by ZQlby** [reply]
not work, i still cant open up the button in securty-login-email there...
![1](https://github.com/outline/outline/assets/28595778/8e338ff0-6830-4b9a-98f0-b9c03bd5afa5)

**Comment by aridwan** [comment]
```
SMTP_HOST=smtp.gmail.com
SMTP_PORT=587
SMTP_USERNAME=username@gmail.com
SMTP_PASSWORD=abcd efgh ijkl mnop
SMTP_FROM_EMAIL=username@gmail.com
SMTP_REPLY_EMAIL=
SMTP_TLS_CIPHERS=TLSv1.2
SMTP_SECURE=false
```

worked well for me

**Comment by telnetdoogie** [comment]
I have tried so many different options for SMTP configuration and none of them have worked. I'm using Gmail. I've tried with the "known service" of Gmail, I've tried the above @aridwan option - with spaces in the app password, no spaces in the app password, I've tried just about everything I can think of, but I still get 

```
{"error":"Invalid login: 535-5.7.8 Username and Password not accepted. For more information, go to
535 5.7.8  https://support.google.com/mail/?p=BadCredentials 586e51a60fabf-3d1e20f203dsm2502539fac.4 - gsmtp",
"level":"error",
"message":"Error sending email to xxx@xxx.com",
"stack":"Error: Invalid login: 535-5.7.8 Username and Password not accepted. For more information, go to
535 5.7.8  https://support.google.com/mail/?p=BadCredentials 586e51a60fabf-3d1e20f203dsm2502539fac.4 - gsmtp
    at SMTPConnection._formatError (/opt/outline/node_modules/nodemailer/lib/smtp-connection/index.js:809:19)
    at SMTPConnection._actionAUTHComplete (/opt/outline/node_modules/nodemailer/lib/smtp-connection/index.js:1595:34)
    at SMTPConnection.<anonymous> (/opt/outline/node_modules/nodemailer/lib/smtp-connection/index.js:556:26)
    at SMTPConnection._processResponse (/opt/outline/node_modules/nodemailer/lib/smtp-connection/index.js:993:20)
    at SMTPConnection._onData (/opt/outline/node_modules/nodemailer/lib/smtp-connection/index.js:774:14)
    at SMTPConnection._onSocketData (/opt/outline/node_modules/nodemailer/lib/smtp-connection/index.js:195:44)
    at TLSSocket.emit (node:events:519:28)
    at TLSSocket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TLSWrap.onStreamRead (node:internal/stream_base_commons:189:23)"
}
```

**Reply by telnetdoogie** [reply]
I'll answer my own question here...

```
SMTP_HOST=smtp.gmail.com
SMTP_PORT=465
SMTP_USERNAME=name@domain.com
SMTP_PASSWORD=app_password
SMTP_FROM_EMAIL=name@domain.com
SMTP_REPLY_EMAIL=name@domain.com
SMTP_TLS_CIPHERS=
SMTP_SECURE=true
```

For me, the key was to completely remove the container and the DB, and re-initialize. I _feel like_ some of the ENV settings that were set were persisting across restarts.

I'm using outline v1 in docker, and Google Workspaces.
